openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/builders/WheelchairGraphStorageBuilder.java

```diff
@@ -201,28 +211,23 @@ private void processSidewalksAttachedToWay(ReaderWay way) {
 	 * way. The same as the attached processing, it looks for the different attributes as tags that are important for
 	 * wheelchair routing and stores them against the generic wheelchair storage object
 	 */
-	private void processWayAsSeparateFeature() {
-
-		setSeparateWayAttribute(WheelchairAttributes.Attribute.SURFACE);
-
-		setSeparateWayAttribute(WheelchairAttributes.Attribute.SMOOTHNESS);
-
-		setSeparateWayAttribute(WheelchairAttributes.Attribute.TRACK);
-
-		setSeparateWayAttribute(WheelchairAttributes.Attribute.WIDTH);
-
-		setSeparateWayAttribute(WheelchairAttributes.Attribute.INCLINE);
-
+	private void processWayCheckForSeparateFeature(ReaderWay way) {
```

Comment:
Should add the new attribute in the JDoc items

Reply:
done